schedule.py

```diff
@@ -0,0 +1,38 @@
+from typing import Iterable
+from flask_restful import abort
+
+from scheduledao import ScheduleDAO
+from event import Event
+
+class Schedule:
+    dao = ScheduleDAO()
+
+    def __init__(self, document: dict):
+        self.owner = document['owner'] if 'owner' in document else None
+        self.events = set(document['events']) if 'events' in document else set()
+
+    def serialize(self):
```

Comment:
didn't go with a list of event_id's ? .. what would happen if an event is modified, do you modify the event in the schedule events list as well? now further coupling the services together?